Sources/_CShims/include/io_shims.h

```diff
@@ -43,4 +43,37 @@ typedef struct FullPathAttributes {
 #pragma pack(pop)
 
 #endif // TARGET_OS_EXCLAVEKIT
+
+#if TARGET_OS_WINDOWS
+
+// Replicated from ntifs.h
```

Comment:
Are we able to just import this header here to expose it directly to swift so that we don't have to copy the definition? (that's what I did with some of the Glibc headers that we could import but weren't part of the modulemap in the toolchain

Reply:
No, I don't think that we should be vending the header, and getting the header path is really complicated.